rules-reviewed/jboss-eap/weblogic/tests/weblogic-xml-descriptors.windup.test.xml

```diff
@@ -0,0 +1,192 @@
+            </when>
+            <perform>
+               <fail message="WebLogic EJB XML descriptor was not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-05000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic EJB XML" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic EJB XML descriptor was not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-06000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic EJB XML" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic EJB XML not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-07000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic SOAP Client" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic SOAP Client classification not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-09000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic Webservice Policy" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic Webservice Policy was not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-10000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic Webservice Descriptor" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic Webservice Descriptor not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-11000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic JMS Descriptor" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic JMS Descriptor not found!" />
+            </perform>
+         </rule>
+         <rule id="weblogic-xml-descriptors-12000-test">
+            <when>
+               <not>
+                  <classification-exists
+                     classification="WebLogic Web Application Descriptor" />
+               </not>
+            </when>
+            <perform>
+               <fail message="WebLogic Web Application Descriptor not found!" />
+            </perform>
+         </rule>
+         <!--  rule id="weblogic-xml-descriptors-13000-test">
```

Comment:
idk, I haven't prepared test data (therefore it is commented) as this whole rule weblogic-xml-descriptors-13000 is not clear to me. Please see https://github.com/mareknovotny/windup-rulesets/blob/xmlrules-review/rules-reviewed/jboss-eap/weblogic/weblogic-xml-descriptors.windup.xml#L222